frontend/app/src/screens/EarnPoolScreen/PanelUpdateDeposit.tsx

```diff
@@ -117,8 +117,8 @@ export function PanelUpdateDeposit({
                 <Tabs
                   compact
                   items={[
-                    { label: "Deposit", panelId: "panel-deposit", tabId: "tab-deposit" },
-                    { label: "Withdraw", panelId: "panel-withdraw", tabId: "tab-withdraw" },
+                    { label: "Increase Deposit", panelId: "panel-deposit", tabId: "tab-deposit" },
+                    { label: "Decrease Deposit", panelId: "panel-withdraw", tabId: "tab-withdraw" },
```

Comment:
To be consistent with the other buttons:

```suggestion
                    { label: "Increase deposit", panelId: "panel-deposit", tabId: "tab-deposit" },
                    { label: "Decrease deposit", panelId: "panel-withdraw", tabId: "tab-withdraw" },
```

Reply:
Good idea yes, that makes this field aligned with the collateral field of the “Update loan” panel :+1: